frontend/jupyter/cypress/fixtures/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "namespace": "kubeflow-user",
+  "postHeaders": {
+    "X-XSRF-Token": "asdf",
```

Comment:
Hardcoded token? 
I might not be understanding the purpose of this file exactly

Reply:
Honestly, this was copy pasted from upstream. And looking at it, the postHeaders isnt even being used so i can just remove it.